src/body.jsx

```diff
@@ -0,0 +1,48 @@
+import "./styles/body.css";
+import PostForm from "./postform";
+function Body() {
+  return (
+    <div className="container">
+      <div className="side-left">
+        <div className="home-icon">
+          <a href="#">
+            <svg
+              width="33"
+              height="36"
+              viewBox="0 0 33 36"
+              fill="none"
+              xmlns="http://www.w3.org/2000/svg"
+            >
+              <path
+                d="M0.5 36V12L16.6 0L32.5 12V36H20.8V21.75H12.15V36H0.5Z"
+                fill="#64648C"
+              />
+            </svg>
+            Home
+          </a>
+        </div>
+        <div className="profile-icon">
+          <a href="#">
+            <svg
```

Comment:
you should turn the SVGs in their separate components, to make it easier to read this file